### ISSUE TITLE
Documentation for dynamic SONG

### ISSUE THREAD

**Issue by alekspejovic** [overture-stack/SONG#443]
- Provide written instructions for setting up new dynamic SONG and instructions for the allowed types
- Need to update read the docs with dynamic schema feature description, as well add to the docker for song tutorial

In addition:
- update production deployment instructions for users. No longer providing tarballs, just docker image. If user wants bare-metal, they can pull the image, and extract the executable.

**Comment by rtisma** [comment]
depends on #542

**Comment by rosibaj** [comment]
This is a duplicate ticket; i am completing this in scope of another ticket. 

closing.
